test/integration/etcd_storage_path_test.go

```diff
@@ -437,7 +437,7 @@ var etcdStorageData = map[schema.GroupVersionResource]struct {
 	// --
 
 	// k8s.io/kubernetes/pkg/apis/autoscaling/v2alpha1
-	gvr("autoscaling", "v2alpha1", "horizontalpodautoscalers"): {
+	gvr("autoscaling", "v2alpha1", "horizontalpodautoscalers"): { // TODO should we be able to use this endpoint at all?
```

Comment:
There is post-rebase task in https://github.com/openshift/origin/issues/13732: "disable alpha APIs" that @mfojtik is taking care of. So this gv will probably go away.